language-design.md

```diff
@@ -74,7 +104,9 @@ the commenters?
 
 **Response**
 
+The comments took on a variety of tones. Some programmers want to consider themselves as [special snowflakes](http://i.imgur.com/AfnKFeA.jpg) with superior intellects. For example, one guest user claims that the reason some people can’t understand the syntax is because, “Most likely because their brains do not process information like a real programmer does.” [Anonymous Guest](https://www.fastcodesign.com/1665735/why-arent-computer-programming-languages-designed-better#comment-2dfe2900-7795-11e2-9384-9794fc737ae8). However, other people had more reasonable approaches.
 
+One point several commenters brought up is that any field that involves careful precision will involve difficult-to-use tools. One user, named Strengthness, compared it surgery, referencing how a surgeon would need lots of training to understand a complex system not easily graspable by the outside layman [Strengthness](https://www.fastcodesign.com/1665735/why-arent-computer-programming-languages-designed-better#comment-93e75ca0-9b5b-11e4-a3e4-4930d33952b2). This is a valid comparison, in that we do expect highly technical fields to require tools that may seem illogical to the non-technical user. However, what many commenters failed to address was that the quasi-illegible syntax creates a higher barrier to entry for aspiring programmers. Programming should be more accessible, not elitist. 
```

Comment:
We also discussed the debate of how accessible a programming language should be to outside users. The commenters bring up an interesting point that other fields need specialized training, so why should it be any different for computer scientists? I would have liked to hear more of your opinions on how programming languages can range from easy to difficult. I agree that some languages are very hard for the aspiring programmer to grasp, but should all languages sacrifice efficiency for accessibility?